tests/contrib/grpc/test_grpc.py

```diff
@@ -149,6 +157,34 @@ def test_pin_can_be_defined_per_channel(self):
         channel1.close()
         channel2.close()
 
+class GrpcServerTestCase(GrpcServerBaseMixin, unittest.TestCase):
+
+    def test_server_span_has_correct_parent_id(self):
```

Comment:
A doctring here describing what this test is doing would be helpful, it took me a minute to figure out why we even had this test case.

Same goes for most other test cases too, we should probably have at least a quick one-liner description.